spec/views/collections/_collection.html.erb_spec.rb

```diff
@@ -18,6 +18,11 @@
     expect(rendered).to have_css('h2 a', text: collection.name)
   end
 
+  it 'renders a paragraph with the description of the collection' do
+    render
+    expect(rendered).to have_css('p', text: collection.description)
+  end
+
```

Comment:
Hmm, interesting.  What would a test for the description not rendering in the partial when there is no description there look like?